clients/fides-js/src/components/DataUseToggle.tsx

```diff
@@ -17,6 +17,7 @@ const DataUseToggle = ({
   disabled,
   isHeader,
   includeToggle = true,
+  secondToggle,
```

Comment:
this is not a good pattern! it's weird that there'd be a toggle defined in the component, and a separate one passed in. but it works quite well, and time is short 🥲 

eventually, I think we should totally rework `TcfVendors` to be a table, and maybe not even use `DataUseToggle` which was never really meant to be a table.

Reply:
follow up ticket: https://github.com/ethyca/fides/issues/4190